xmtp_mls/src/groups/mod.rs

```diff
@@ -272,6 +272,15 @@ where
         account_addresses_to_add: Vec<String>,
     ) -> Result<(), GroupError> {
         let account_addresses = sanitize_evm_addresses(account_addresses_to_add)?;
+        // get current number of users in group
+        let member_count = self.members().unwrap().len();
```

Comment:
Can you use `?` here rather than `unwrap()`, to forward the error rather than panicking?